app/controllers/mixins/generic_show_mixin.rb

```diff
@@ -128,7 +148,7 @@ def nested_list(table_name, model, options = {})
       drop_breadcrumb(:name => _("%{name} (All %{title})") % {:name => @record.name, :title => title},
                       :url  => show_link(@record, :display => @display))
 
-      view_options = {:parent => @record}
+      view_options = {:parent => @record, :association => options[:association]}
```

Comment:
Glancing at the code of `get_view` I'd say that the result is the same so I'd prefer to have both options be handled the same way. But this is a minor issue not worth too much effort.